versions.tf

```
@@ -2,6 +2,11 @@ provider "aws" {
   region = "eu-west-2"
 }
 
+provider "aws" {
+  alias  = "org-cert"
+  region = "us-east-1"
```

SInce this isn't obvious, maybe worth adding a comment, e.g.
```suggestion
  # Cloudfront only accepts certificates in us-east-1
  region = "us-east-1"
```